content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -25,6 +26,8 @@ _Topology Manager_ is a Kubelet component that aims to co-ordinate the set of co
 {{% capture prerequisites %}}
 
 {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+In Kubernetes versions 1.16 and 1.17, the Topology Manager feature gate must be enabled manually via the Kubelet `--feature-gates` flag.
+Please see documentation related to the Topology Manager [`feature gate`](#topology-manager-kubelet-feature-gate) for more details.

```

Comment:
I'm not keen on this. The text “feature gates” should be a hyperlink to https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/ - see other pages for how this is typically done.

Given the `{{< version-check >}}` shortcode, it's OK to assume that readers have the feature enabled.